examples/distributedLock/README.md

```diff
@@ -0,0 +1,78 @@
+# Example - Distributed Lock APIs
```

Comment:
Given that in both examples we have both `tryLock` and `unlock` functions being invoked, is it possible to merge the two examples into one by taking a specific scenario and demonstrate both?